### DISCUSSION TITLE
015: Exploring Turbo 8 with Jorge Manrubia

### DISCUSSION THREAD

**Discussion by siaw23** [therubyradar/submissions#191] [Submissions]
[In this episode](https://share.transistor.fm/s/0246808b), Jorge Manrubia provides insights into his experience working at 37Signals. Our conversation delves into the intricacies of Active Record encryption, and we explore the latest advancements in Turbo 8 technology.

**Comment by cjilbert504** [comment]
Thanks @siaw23!!!!

**Reply by siaw23** [reply]
No, thank **YOU**! 😀
